semantics/src/main/scala/aqua/semantics/rules/ValuesAlgebra.scala

```diff
@@ -99,7 +99,7 @@ class ValuesAlgebra[S[_], Alg[_]: Monad](using
           case None =>
             (for {
               t <- OptionT(
-                T.getType(name.value)
+                T.getType(name.value, name)
```

Comment:
We could accept `NamedTypeToken[S]` in `getType`. `Name[S]` have method `asTypeToken`

Reply:
Also `getType` does exactly the same as `resolveType`, except that `resolveType` might report an error. In `NamesAlgebra` in similar situation we added a boolean flag to control error reporting. Maybe do the same here and remove `getType` method?